src/pages/community/faq.md

```diff
@@ -82,3 +82,18 @@ BuckleScript is optimized for performance across the whole stack. You can try sl
 - Adding a few infinite loops here and there.
 - Stuffing a JavaScript build tool in the pipeline.
 - Dragging in more dependencies for writing a hello world.
+
+### I'm seeing some weird .cm[ixja] files referenced in a compiler error. Where do these files come from?
+
+OCaml outputs a few different files during compilation, depending on your build target (native/bytecode/JavaScript):
+
+.cmi: Compiled interface (.rei/mli) file
```

Comment:
this won't render well - could you prefix each line with `- ` to make it a markdown list?